go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/shirou/gopsutil v3.21.11+incompatible
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.9.0
-	go.opentelemetry.io/collector/featuregate v1.5.0
```

Comment:
Are these gomod changes necessary here?

Reply:
It shouldn't happen in an unrelated PR. #3028 will take care of the featuregate package. Can you also move the one dependency you're adding to the top require section?